data/transport missions.txt

```diff
@@ -1975,3 +1975,44 @@ ship "Science Drone"
+	deadline
+	passengers 15
+	minor
+	to offer
+		random < 25
+		"unpaid mortgages" == 0
+	source "New Boston"
+	destination
+		government "Syndicate"
+	on offer
+		conversation
+			`As you're walking through the spaceport searching for a bar or pub, you notice one person who stands out. A nearly bald, middle-aged man wanders through the crowd in an expensively tailored suit, searching the faces that pass by him. You pause and look at him more closely; he seems oddly familar. Eventually, his gaze settles on you, making eye contact. He stares, then begins moving through the crowd toward you.`
+			choice
+				`	(Wait for him to reach you.)`
+				`	(Walk forward to meet him.)`
+				`	(Turn the other way and lose the man in the crowd.)`
+					decline
+			`	When you and the man come face to face, he studies you for a moment. You realize who he is when you hear his voice: "Captain <last>, correct? I suppose I should formally introduce myself, even though we've met before. I'm Mr. Johnson," he says, extending a hand. You shake it, wondering how the loan broker who lent you money to purchase your first ship could possibly recognize you. "Before you ask, I pay people to keep tabs on all my clients, new and old. I find that it can pay off to have contacts from several different walks of life. Since I was informed a ship registered in your name was in the New Boston spaceport, I decided it was time for a reunion," he explains.`
+			`	 "I've found myself with a problem, and you can assist in solving it. One of my... 'clients' hasn't been very proactive in paying their dues on time. This wouldn't be a problem for any normal client, but this man happens to be in charge of a small street gang on <destination>. Of course, there's a simple fix: send a group of paid enforcers to convince the client to be more... accountable, let's say."`
+			choice
+				`	"So what do I have to do with this?"`
+				`	"Sorry, but I'm too busy to start another job right now."
+					decline
+			`	"Usually I have a different contact that I could use for this kind of job, but she seems to be unavailable at the moment. So, I need a ship to carry my fifteen men to <destination>. I've been told you're the kind of person who might be interested in such a job," Mr. Johnson concludes, looking at you expectantly.`
+			choice
+				`	"Alright, I'll do it."`
+				`	"I don't stoop low enough to carry thugs for a loan shark."`
+					decline
+				`	"Sorry, but I don't have the room to carry them right now."`
+					decline
+			`	He grins. "Wonderful. Efficiency and speed are important to me, Captain <last>, and you will be rewarded for it. Don't worry about the legality of your part of the job; my men won't be "on the clock" until they are off your ship and out of whatever spaceport you land at. One of my men on <destination> will see to it that you are well compensated."`
+				accept
+	on complete
+		payment 400000
```

Comment:
That's an awful lot of money for a glorified transport mission, especially considering that it doesn't even spawn pirates or bounty hunters or involve any additional risk to the player.

Is this guy going to be able to make any profit off the deal after he's paid you? 😛